STYLEGUIDE.md

```diff
@@ -0,0 +1,101 @@
+# Spyder Documentation Style Guide:
+
+
+## General
+
+Must (not), should (not), recommended, suggested, may, and can generally follow IETF conventions, with some additional differentiation:
+
+- **Must** (not) clauses are hard requirements that must always be observed
+- **Should** (not) clauses can only be disregarded in a particular instance with strong justification, or if it is demonstratively highly impractical to follow for a specific case
+- (Not) **Recommended** and **Preferred** clauses should be followed where practicable; a brief rationale should be provided for not doing so
+- **Suggested** clauses indicate a convention, approach or guideline that may be helpful to follow, but has no binding effect or justification requirements
+- **May** or **Can** clauses designate explicitly permitted guidelines, actions or conventions, often for clarity, that are allowed but have no weight of preference on them (and may even be gently discouraged with a suggested or preferred alternative)
+
+
+## Structure
+
+- In general, follow the [OpenStack Heading Guidelines](https://docs.openstack.org/doc-contrib-guide/writing-style/general-writing-guidelines.html) with regard to sections and headings; in particular, section and subsection titles delineating overall categories of tasks/topics and be in gerund (i.e. with "ing", e.g. "Editing code", "Debugging files"), while those concerning a concept or reference topic should be a noun (e.g. "General considerations", "Requirements") and lowest level topic/task titles should generally be in imperative ("Create a code cell", "Get help", etc) where reasonable
+- Heading case: Page titles (L1 headings) should use Title Case, all others should use Sentence case
+- 1 L1 Heading for page titles: Use ``######`` with overline; on first line of file; keep to a few words
+- L2 Headings for section heads: Use ========= with overline; 3 blank lines before them, will appear in sidebar so keep them short
+- L3 headings for subheads: Use ~~~~~~, 2 blank lines,
```

Comment:
```suggestion
- L3 headings for subheads: Use ~~~~~~, 2 blank lines
```

Good catch. There is a lot of sloppiness here since I never did the big polish pass I was going to.